docs/docsite/rst/inventory_guide/intro_inventory.rst

```diff
@@ -587,7 +587,7 @@ ansible_connection
 General for all connections:
 
 ansible_host
-    The name of the host to connect to, if different from the alias you wish to give to it.
+    The name of the host to connect to, if different from the alias you wish to give to it. Never set it to depend on ``inventory_hostname`` unless you plan to never use delegation.
```

Comment:
```suggestion
    The name of the host to connect to, if different from the alias you wish to give to it. Never set it to depend on ``inventory_hostname`` if you use delegation.
```

Reply:
Hopefully this doesn't change meaning but the original wording is a bit hard to grok.

Reply:
it was 'rage induced PR' so i might not have been in the best place to make the wording clear. Thanks for suggestion!